app/containers/Industry/InputHook.js

```diff
@@ -0,0 +1,17 @@
+import {useState} from 'react';
+
+export const useInput = initialValue => {
```

Comment:
I am not sure what this file is supposed to do. Can you add come comments please?

Reply:
sure, it's just a custom hook to handle form input